.github/workflows/int-test-workflow.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - develop
       - main
+      - prodsec-fix
```

Comment:
will we continue to include prodsec-fix branch in the tests?

Reply:
once everything passses, will remove this line.